### DISCUSSION TITLE
Installing SEB on a Mac running a Windows OS

### DISCUSSION THREAD

**Discussion by uqswalk5** [SafeExamBrowser/seb-win-refactoring#505] [Q&A]
Hi team SEB,

I have a student running Windows OS on a Mac. We get a lot of asian students who own Macs but never use the Mac OS preferring to always use Windows as their OS. In the past they have just installed SEB for Windows and had no problems. Is this your understanding of how this should work?

This particular student has a problem installing and has received this message.  
https://user-images.githubusercontent.com/29533145/200686345-500cbcff-380e-457a-9672-3f9f18c10a87.png

I have asked him to send me the logs from the pathway shown in the message but he cannot find them! If SEB were installing on a Mac running Windows OS is this where you would expect the logs to be?

C:\Users\apple\AppData\Local\SafeExamBrowser\Logs

Thanks for all the hard work and the excellent product.

**Comment by dbuechel** [comment]
> Is this your understanding of how this should work?

There is no obvious reason why this shouldn't work, apart from the VM detection which might get activated when running SEB on a virtualized Windows on macOS.

> If SEB were installing on a Mac running Windows OS is this where you would expect the logs to be?
> C:\Users\apple\AppData\Local\SafeExamBrowser\Logs

Yes, if that's the path shown in the error message, then the logs should be saved there. It does appear that SEB was correctly installed, but seems to be unable to start a new session correctly (though I don't speak Chinese so I can't determine the exact phrase stated in the error message). Your user may try to copy and paste `%LocalAppData%\SafeExamBrowser\Logs` in the address bar of a Windows Explorer instance, this should get them right to the log files.
